game/src/GameMenu.tscn

```diff
@@ -1,10 +1,9 @@
-[gd_scene load_steps=7 format=3 uid="uid://b4pg2y2ivib8f"]
+[gd_scene load_steps=6 format=3 uid="uid://vbtt55n875k1"]
 
 [ext_resource type="Script" path="res://src/GameMenu.gd" id="1_cafwe"]
 [ext_resource type="PackedScene" uid="uid://dvoin538iby54" path="res://src/MainMenu/MainMenu.tscn" id="2_2jbkh"]
 [ext_resource type="PackedScene" uid="uid://cnbfxjy1m6wja" path="res://src/OptionMenu/OptionsMenu.tscn" id="3_111lv"]
-[ext_resource type="PackedScene" uid="uid://b7oncobnacxmt" path="res://src/LocaleButton.tscn" id="4_jno35"]
```

Comment:
I mean, it makes no difference, if they don't know any of the language they're not going to be able to find the locale button on the main menu rather than the options menu, and assuming we ship the manual in multiple languages, one of the first sections can simply be where the options menu is and how to change the language.

Reply:
That's requiring excessive steps and other people involved, which is dumb, nobody should require outside help just to start the game, a locale button at start will eliminate this if we put an icon for each language name.